migrations/2024-11-27-143958_add_transaction_flow_in_payment_method/up.sql

```diff
@@ -0,0 +1,2 @@
+-- Your SQL goes here
+ALTER TABLE payment_methods ADD COLUMN IF NOT EXISTS transaction_flow VARCHAR(255);
```

Comment:
This seems as an enum in our diesel model, any specific reason we aren't creating a type for this in DB ?

Reply:
done